nav2_bringup/launch/tb3_simulation_launch.py

```diff
@@ -43,6 +46,7 @@ def generate_launch_description():
     autostart = LaunchConfiguration('autostart')
     use_composition = LaunchConfiguration('use_composition')
     use_respawn = LaunchConfiguration('use_respawn')
+    use_gz = LaunchConfiguration('use_gz')
```

Comment:
Remove this work and make a new launch file for new-gazebo support so we have both in parallel. This is going to help us to migrate forward for having different robots / environments whereas we don't want to be constantly bloating this file